native/cocos/platform/SDLHelper.cpp

```diff
@@ -335,44 +342,50 @@ void SDLHelper::pollEvent(bool *quit) {
     }
 }
 
-bool SDLHelper::createWindow(const char *title,
+SDL_Window *SDLHelper::createWindow(const char *title,
                              int w, int h, int flags) {
-    if (_isWindowCreated) {
-        return true;
-    }
+    //if (_isWindowCreated) {
+    //    return true;
+    //}
     SDL_Rect screenRect;
     if (SDL_GetDisplayUsableBounds(0, &screenRect) != 0) {
-        return false;
+        return nullptr;
     }
     int x = screenRect.x;
     int y = screenRect.y + screenRect.h - h;
     return createWindow(title, x, y, w, h, flags);
 }
 
-bool SDLHelper::createWindow(const char *title,
+SDL_Window *SDLHelper::createWindow(const char *title,
                              int x, int y, int w,
                              int h, int flags) {
-    if (_isWindowCreated) {
-        return true;
-    }
+    //if (_isWindowCreated) {
```

Comment:
If you comment out here, you need to pay attention to the platforms of linux and QNX, which are common to multiple platforms